validation/citrus-validation-json/src/main/resources/META-INF/citrus/message/schemaValidator/jsonSchema

```diff
@@ -0,0 +1,2 @@
+name=defaultJsonSchemaValidator
```

Comment:
I wonder weather the file should be named `json` instead of `jsonSchema` so someone can lookup the validator by its name easily. Also then someone could get the default validator for the message type `json` by name. WDYT?